gr-blocks/lib/correctiq_auto_impl.cc

```diff
@@ -0,0 +1,123 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2020 Free Software Foundation, Inc.
+ *
+ * This file is part of GNU Radio
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ *
+ */
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include "correctiq_auto_impl.h"
+#include <gnuradio/io_signature.h>
+
+namespace gr {
+namespace blocks {
+
+correctiq_auto::sptr
+correctiq_auto::make(float samp_rate, float freq, float gain, float syncWindow)
+{
+    return gnuradio::get_initial_sptr(
+        new correctiq_auto_impl(samp_rate, freq, gain, syncWindow));
+}
+
+/*
+ * The private constructor
+ */
+correctiq_auto_impl::correctiq_auto_impl(float samp_rate,
+                                         float freq,
+                                         float gain,
+                                         float syncWindow)
```

Comment:
:camel:

Reply:
@mbr0wn dromedary?

Reply:
Camel Case variable. `syncWindow` should be `sync_window`

Reply:
Lol, yeah I thought I search/replaced 'em all out.  Guess I missed one.  Old training.... snake_case was legacy c code and embedded, and anything OO was to be camel.  If you look at gr-correctiq, I caught most of them! :)